src/Groups/GAPGroups.jl

```diff
@@ -2217,22 +2217,23 @@ function _map_word_syllable(vi::Int, genimgs::Vector, genimgs_inv::Vector)
   vi = -vi
   @assert vi <= length(genimgs)
   isassigned(genimgs_inv, vi) && return genimgs_inv[vi]
-  res = inv(genimgs[vi])
-  genimgs_inv[vi] = res
-  return res
+  genimgs_inv[vi] = one(genimgs[1])
+  genimgs_inv[vi] = inv!(genimgs_inv[vi], genimgs[vi])
```

Comment:
```suggestion
  genimgs_inv[vi] = inv(genimgs[vi])
```
this does the same thing, but more efficient.
you currently allocate a one object to then modify it to contain an inverse. But just allocating an inverse could be faster (or in the worst-case fall back to the before)